### DISCUSSION TITLE
626690-Sword Art Online: Fatal Bullet

### DISCUSSION THREAD

**Discussion by Namjw11** [pjy612/SteamManifestCache#79] [Share]
`appid:626690`
`key：`
`626691;d13786cc9fbb38ba37c0da1e5a7148f908520d248bc`
code：BE778A96CA24DDDB
email：namjw11@hotmail.com
[626690_config.vdf.txt](https://github.com/pjy612/SteamManifestCache/files/13696463/626690_config.vdf.txt)

**Comment by pjy612** [comment]
查收

**Reply by Namjw11** [reply]
怎么我的内测码会自己变呀。。我昨天用fix2也显示报错，今天用fix3准备试试发现code变了。。。能给我再发一份吗现在的code是

`0564F230E04289F2`

**Reply by pjy612** [reply]
你是不是 系统设置 WLAN 啥的 开了 随机硬件地址？ 那个会影响码的计算。如果在家的话 对自己家的 wifi 地址 选择 关。

**Reply by Namjw11** [reply]
没有开启，但是我找到原因了。我把梯子的tun模式打开又变回去了
